tools/mirror/src/lib.rs

```diff
@@ -234,6 +286,22 @@ fn get_last_source_height(db: &DB) -> anyhow::Result<Option<BlockHeight>> {
         .map(|v| BlockHeight::try_from_slice(&v).unwrap()))
 }
 
+fn get_first_accounts_created(db: &DB) -> anyhow::Result<bool> {
```

Comment:
what does creating first accounts mean? is it about the initial setup? can you add comments to clarify these?